qemu/tests/set_link.py

```diff
@@ -195,6 +195,8 @@ def set_link_test(linkid, link_up, expect_status,
     change_queues = False
     guest_ifname = ""
     guest_ip = vm.get_address()
+    host_ip = utils_net.get_ip_address_by_interface(params.get("netdst"))
```

Comment:
why use srchost,  it looks srchost only used for migrate?
I think is enough.
host_ip  = utils_net.get_host_ip(params)